rep-2001.rst

```diff
@@ -773,6 +775,33 @@ Desktop Full
       packages: [perception, simulation, ros_ign_gazebo_demos]
 
 
+ROS Distribution Independent (September 2022 - Ongoing)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These variants are not released as ROS packages within each distribution as the others are.
+Instead they are made available as independent packages outside of any ROS distribution.
+
+Build Essential
+"""""""""""""""
+
+::
+
+ - ros-build-essential:
+      packages: [cmake, git, python3-colcon-common-extensions, python3-rosdep,
+                 python3-setuptools, wget]
+
+Dev Tools
+"""""""""
+
+::
+
+  - ros-dev-tools:
+      extends:  [ros-build-essential]
+      packages: [python3-flake8, python3-pip, python3-pytest,
+                 python3-pytest-cov, python3-vcstool]
```

Comment:
```suggestion
  - ros-dev-tools:
      extends:  [ros-build-essential]
      packages: [python3-bloom, python3-colcon-common-extensions,
                 python3-rosdep, python3-vcstool]
```

We discussed this on an earlier revision https://github.com/ros-infrastructure/rep/pull/355#discussion_r908982708 but I don't think the pytest or flake8 packages belong here. I think that those should be per-package dependencies and moved out of the documentation where they occur.

The original reason they ended up there was because not all of the flake8 or pytest extensions used were available as apt packages in ROS 2's target platforms. They're available as-of Jammy and so we'll have to decide if it's worth adding them in dev-tools for the sake of ROS 2 Foxy.

I should have considered this as an explicit case for how to handle differences between distros for independent metapackages but I didn't because I don't think they belong in this metapackage for any distro in the first place.